lib/index.js

```diff
@@ -25,30 +27,28 @@ var formatParsedObject = function(type, input, valid, parsed) {
 };
 
 /* istanbul ignore next */
-Utils._getCurrentDate = function() {
+Utils._getCurrentDate = function(options) {
   // Makes unit testing possible, by allowing this
   // function to be mocked.
-  return new Date();
+  return SugarDate.create('now', options);
 };
 
 /* istanbul ignore next */
-Utils._getFutureDate = function(str) {
+Utils._getFutureDate = function(str, options) {
   // Makes unit testing possible, by allowing this
   // function to be mocked.
-  return Date.future(str);
+  return SugarDate.create(str, _.assign({future: true}, options));
```

Comment:
This would be better written as:

``` js
return SugarDate.create(str, _.assign({}, options, {future: true}));
```

The advantage here is that an `options` object containing `future: false` would not cause issues.